dlgr/griduniverse/experiment.py

```diff
@@ -827,11 +860,9 @@ def rank(self, color):
 class Food(object):
     """Food."""
     def __init__(self, **kwargs):
-        super(Food, self).__init__()
```

Comment:
Someone (@suchow?) has a macro in their code editor which adds these stub `super()` calls to `__init__()` methods, so they're floating around all over the place.

Reply:
@jessesnyder Is `super` not so super?

Reply:
`object.__init__()` does nothing (I researched this when I first saw these calls, wondering if I was missing something: https://hg.python.org/cpython/file/2.7/Objects/typeobject.c#l2938), so there's no reason to call it.